_data/projects/Numbas.yml

```diff
@@ -0,0 +1,19 @@
+name: Numbas
+desc: An e-assessment system aimed at mathematical disciplines. It produces SCORM packages - everything happens on the client in JavaScript! The package compiler is written in Python. There's plenty for someone with a good knowledge of maths to work on.
+site: http://www.numbas.org.uk/contributing-to-numbas/
+tags:
+- python
+- javascript
+- e-assessment
+- SCORM
+- math
+- web
+- HTML
+- CSS
+- html5
+- algebra
+- CAS
+- quiz
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/numbas/Numbas/issues?q=is%3Aopen+is%3Aissue+label%3Aup-for-grabs
```

Comment:
For a bit of :lipstick: you can use `https://github.com/numbas/Numbas/issues/labels/up-for-grabs` here